### DISCUSSION TITLE
only want to use point and distance how do I do it? tried with @turf/distance, but no point in it

### DISCUSSION THREAD

**Discussion by joorraa** [Turfjs/turf#2387] [General]
Any recommendation? I don't want to use @turf/turf since its huge.

**Comment by rowanwins** [comment]
@mainaani it's a bit unclear what are wanting to achieve - which functions are you wanting, and why aren't you able to import them the  relevant module?

**Reply by joorraa** [reply]
@rowanwins mate I am basically just trying achieve example shown in @turf/distance: https://www.npmjs.com/package/@turf/distance

```
var from = turf.point([-75.343, 39.984]);
var to = turf.point([-75.534, 39.123]);
var options = {units: 'miles'};

var distance = turf.distance(from, to, options);

//addToMap
var addToMap = [from, to];
from.properties.distance = distance;
to.properties.distance = distance;
```

I only want to use point and distance function. want to use it individually. noy being able to import { point, distance } from '@turf/distance';

**Reply by stebogit** [reply]
@mainaani the `point` function [comes from](https://turfjs.org/docs/#point) the `@turf/helpers` package.
`@turf/distance` only contains the `distance` function

**Reply by rowanwins** [reply]
To add to @stebogit 's answer you should be able to do
`npm install @turf/helpers @turf/distance`
followed by
````
import {point} from '@turf/helpers'
import distance from '@turf/distance'
````

**Reply by joorraa** [reply]
@stebogit @rowanwins  thanks you good sirs
